pyswarms/single/local_best.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, n_particles, dimensions, options, bounds=None, velocity_clamp
             a tuple of size 2 where the first entry is the minimum velocity
             and the second entry is the maximum velocity. It 
             sets the limits for velocity clamping. 
-        options : dict with keys :code:`{'c1', 'c2', 'k', 'p'}`
+        options : dict with keys :code:`{'c1', 'c2', 'w', 'k', 'p'}`
```

Comment:
Thanks for catching this! 👍

Reply:
No problem! 😄 

It's been great to read the documentation on this project, as it's super clear, and I'm glad you're setting a nice standard for that.